pyarlo/base_station.py

```diff
@@ -113,7 +301,11 @@ def mode(self, mode):
         """
         if mode not in ACTION_MODES.keys():
             return "Invalid mode"
-        return self.__run_action(mode)
+        return self.__run_action(
+            method='SET',
+            resource='modes',
+            mode=mode,
+            publish_response=True)
```

Comment:
If really required, we can call them in the mode setter function after running the __run_action() to be sure.

That way, we don't have to do that update explicitly.  When someone sets the mode, it gets updated automatically by the library itself.